packages/autorest.python/autorest/codegen/serializers/builder_serializer.py

```diff
@@ -58,6 +58,14 @@
 )
 
 
+def _xml_config(send_xml: bool, content_types: List[str]) -> str:
+    if not (send_xml and "xml" in str(content_types)):
+        return ""
+    if len(content_types) == 1:
+        return ", is_xml=True"
+    return ", is_xml=kwargs.pop('is_xml', 'xml' in str(content_type))"
```

Comment:
we don't have an "is_xml" kwarg that we accept, I'm ok to just switch this to "xml" in str(content_type)

Reply:
i'll update the code and you can let me know if it's good to release on Monday. I think this will fix the apimanagement issue but i want to verify with you first

Reply:
It is enough to fix the issue for apimanagement so please release new version.